src/stream/src/executor/over_window/over_partition.rs

```diff
@@ -425,6 +425,8 @@ impl<'a, S: StateStore> OverPartition<'a, S> {
         Option<RangeInclusive<StateKey>>,
     )> {
         let input_schema_len = table.get_data_types().len() - calls.len();
+        let rank_funcs_only = calls.iter().all(|call| call.kind.is_rank());
```

Comment:
Seems this is static? We can do this check once when instantiating the executor.

Reply:
Yes. Let me eliminate this reevaluation in a separate PR, cuz we also have other things that are evaluated each time when creating `OverPartition` in `OverPartition::new`. To do this, I need to change some code unrelated to the optimization proposed in this PR.